<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileErrorDialog</class>
 <widget class="QWidget" name="fileErrorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error with file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Error</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_error">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_size">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_content_size">
         <property name="text">
          <string notr="true">0 KiB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_modified">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modified</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_content_modified">
         <property name="text">
          <string notr="true">Today</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_file_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_content_file_name">
         <property name="text">
          <string notr="true">source.txt</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_file_destination">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Destination</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_content_file_destination">
         <property name="text">
          <string notr="true">../toto.txt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_folder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_content_folder">
         <property name="text">
          <string notr="true">/folder/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxAlways">
       <property name="text">
        <string>&amp;Always perform this action</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Rights">
       <property name="text">
        <string>Try in with elevated privileges</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PutToBottom">
       <property name="text">
        <string>Put to bottom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Retry">
       <property name="text">
        <string>Retry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Skip">
       <property name="text">
        <string>&amp;Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>toggled(bool)</signal>
   <receiver>Cancel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>toggled(bool)</signal>
   <receiver>Retry</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
